### DISCUSSION TITLE
[持续更新] Hexo-Theme-Type 最新进展

### DISCUSSION THREAD

**Discussion by aiokr** [aiokr/Tripper-Press#23] [Announcements]
**近期规划请浏览这里：[[持续更新] Hexo-Theme-Type 近期规划 · Discussion #24 · aiokr/Tripper-Press](https://github.com/aiokr/Tripper-Press/discussions/24)**

## 2022年4月最新进展

- [@079f123](https://github.com/aiokr/Tripper-Press/commit/079f1237cda80428aa36c23475cda59b75e4df82) 新增了独立的相册页面
- [@9a8d4d3](https://github.com/aiokr/Tripper-Press/commit/9a8d4d3b416726169a222b5e3fa15337866d41fb) 给 M3 样式的 Waline 评论系统设计了一套主题。
- [@a055258](https://github.com/aiokr/Tripper-Press/commit/a055258b2aebf2d44427a04ba60d929e1026f916) 新增了一套 M3 样式，部分依照 Material Design 3 风格进行设计，原有样式 Column 保留，并将会继续维护。

## 2022年3月最新进展

- [@1881ff5](https://github.com/aiokr/Tripper-Press/commit/1881ff5457197b03ec8b2e61603aa7420e568f2d) 通过脚本自动打包主题到[仓库](https://github.com/aiokr/hexo-theme-type)，并保留更新记录。

**Comment by ZHU-SK** [comment]
作者你好，距离上次更新已过去接近半年时间，请问是否有后续更新的计划呢？多谢。
